components/nimbus/src/enrollment.rs

```diff
@@ -1318,3 +1384,94 @@ pub(crate) fn now_secs() -> u64 {
         .expect("Current date before Unix Epoch.")
         .as_secs()
 }
+
+#[cfg(test)]
+mod unit_tests {
```

Comment:
Is this unit test module here because of access specification of the `populate_feature_maps` method? Curious if it could be moved into the base `test_enrollment` file.